cves/kernel/CVE-2021-3573.yml

```diff
@@ -305,9 +315,9 @@ discussion:
 
     Put any links to disagreements you found in the notes section, or any other
     comment you want to make.
-  discussed_as_security:
-  any_discussion:
-  note:
+  discussed_as_security: false
+  any_discussion: false
```

Comment:
I believe I found a post discussing your vulnerability here: https://f0rm2l1n.github.io/2021-07-23-Blue-Klotski/ .

Reply:
Thank you I'll look into it and update the yml